packages/webpack-cli/lib/bootstrap.js

```diff
@@ -39,74 +28,45 @@ const runCLI = async (cliArgs) => {
     }
 
     try {
-        // handle the default webpack entry CLI argument, where instead
-        // of doing 'webpack-cli --entry ./index.js' you can simply do
-        // 'webpack-cli ./index.js'
-        // if the unknown arg starts with a '-', it will be considered
-        // an unknown flag rather than an entry
+        // Create a new instance of the CLI object
+        const cli = new WebpackCLI();
+
+        // Handle the default webpack entry CLI argument, where instead of doing 'webpack-cli --entry ./index.js' you can simply do 'webpack-cli ./index.js'
+        // If the unknown arg starts with a '-', it will be considered an unknown flag rather than an entry
         let entry;
-        if (parsedArgs.unknownArgs.length > 0 && !parsedArgs.unknownArgs[0].startsWith('-')) {
-            if (parsedArgs.unknownArgs.length === 1) {
-                entry = parsedArgs.unknownArgs[0];
-            } else {
-                entry = [];
-                parsedArgs.unknownArgs.forEach((unknown) => {
-                    if (!unknown.startsWith('-')) {
-                        entry.push(unknown);
-                    }
-                });
-            }
-        } else if (parsedArgs.unknownArgs.length > 0) {
-            parsedArgs.unknownArgs.forEach((unknown) => {
-                logger.warn(`Unknown argument: ${unknown}`);
+
+        if (parsedArgs.unknownArgs.length > 0) {
+            entry = [];
+
+            parsedArgs.unknownArgs = parsedArgs.unknownArgs.filter((item) => {
+                if (item.startsWith('-')) {
+                    return true;
+                }
+
+                entry.push(item);
+
+                return false;
             });
-            const args = await cliExecuter();
-            const { opts } = parseArgs(args);
-            await cli.run(opts, core);
-            return;
         }
+
+        if (parsedArgs.unknownArgs.length > 0) {
+            parsedArgs.unknownArgs.forEach(async (unknown) => {
+                logger.error(`Unknown argument: ${unknown}`);
+            });
+
```

Comment:
We should point to documentation for all the flags at least here.

Reply:
I think no, we have over 250 flags, we should use levenshtein here to say which option is closer

Reply:
> we should use levenshtein here to say which option is closer
Much better

Reply:
My point we should help user to get things done, if we can't help them through command line, we should at least point to a place where they can look for how to do it.

That being said, using levenshtein is a good option as well.